torch/csrc/jit/tensorexpr/loopnest.cpp

```diff
@@ -681,18 +735,18 @@ Stmt* LoopNest::lowerToStmt(Tensor* t) {
   stmt_to_tensor_[body] = t;
   tensor_to_stmt_[t] = body;
 
-  if (t->buf()->ndim() == 0) {
+  if (f->ndim() == 0) {
```

Comment:
What motivates this change? In the long run I think dimensions should be a property of the tensor (more specifically, of a buffer that tensor refers to), but nor of the function.